apps/routes/lib/route.ex

```diff
@@ -65,7 +65,7 @@ defmodule Routes.Route do
   def type_atom("ferry"), do: :ferry
   def type_atom("909"), do: :logan_express
   def type_atom("983"), do: :massport_shuttle
-  def type_atom("Massport-1"), do: :massport_shuttle
```

Comment:
There's another reference to "Massport-1" in `site_web/views/helpers.ex`.  I'm not sure if that needs to be replaced in the same way, or if `"Massport-" <> _` is even needed because it also checks for :massport_shuttle.

Reply:
Done